package.json

```diff
@@ -11,7 +11,7 @@
     },
     "./runtime": {
       "types": "./runtime/index.d.ts",
-      "style": "./runtime/styles.css",
+      "style": "./runtime/index.css",
```

Comment:
Does it break the backward compatibility?

Reply:
Yep. This will change import on transformer side.
Do you know other broken points?

Reply:
No. I just wanted to highlight it.